.circleci/config.yml

```diff
@@ -1,5 +1,3 @@
-# vim: set expandtab shiftwidth=2 tabstop=8:
-
```

Comment:
why did you remove this one? it's really handy to have the editor do the right thing when opening the file

Reply:
I removed it because it wasn't mentioned in the commit in libratbag that added it and thought it was maybe unintended (also I don't use vim), but I can restore it.

Reply:
it's a useful thing to have. There's a line for emacs too to auto-configure special settings for the opened files but neither @bentiss nor I use emacs, so .. meh :) And these days' there's also [editorconfig](https://editorconfig.org/#example-file) but that's less useful when there's a single file with different requirements.